src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/EditCyclePage/PayoutsSection/PayoutsTable/PayoutsTableBody.tsx

```diff
@@ -8,23 +8,25 @@ import { PayoutSplitRow } from './PayoutSplitRow'
 import { PayoutsTableCell } from './PayoutsTableCell'
 import { PayoutsTableRow } from './PayoutsTableRow'
 import { TotalRows } from './TotalRows'
+import { usePayoutsTableContext } from './context/PayoutsTableContext'
 
 const Row = PayoutsTableRow
 const Cell = PayoutsTableCell
 
 export function PayoutsTableBody() {
+  const { topAccessory } = usePayoutsTableContext()
   const {
     payoutSplits,
     currency,
     handleDeletePayoutSplit,
     setCurrency,
     distributionLimit,
   } = usePayoutsTable()
-
   const emptyState = distributionLimit === 0
 
   return (
     <>
+      {topAccessory}
```

Comment:
does this need to be the `usePayoutsTableContext`? feels a bit weird. wondering if it would work just the same if, in create flow, it was just 
```tsx
<radio buttons>
<PayoutsTableBody/>
```

Reply:
The reason for this was because this radio component relies on the `PayoutsTableContext` which is inside `PayoutsTable`. The best way might be to separate out the context from the `PayoutsTable`, which I thought initially was less clean.